.circleci/config.yml

```diff
@@ -1,32 +1,28 @@
 version: 2
 jobs:
-  build-docker:
+  # Run all tests related to the web client
+  test-webclient:
     docker:
-      - image: circleci/cci-demo-docker-primary:0.0.2
+      - image: circleci/node:latest
     steps:
       - checkout
       - run:
-          name: Install Docker client
+          name: Tests for Web Client
           command: |
-            set -x
-            VER="17.03.0-ce"
-            curl -L -o /tmp/docker-$VER.tgz https://get.docker.com/builds/Linux/x86_64/docker-$VER.tgz
-            tar -xz -C /tmp -f /tmp/docker-$VER.tgz
-            mv /tmp/docker/* /usr/bin
-      - run:
-          name: Install Docker Compose
-          command: |
-            set -x
-            curl -L https://github.com/docker/compose/releases/download/1.25.3/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
-            chmod +x /usr/local/bin/docker-compose
-      - deploy: 
-          name: Build Health Barbecue docker-compose
-          command: |
-            set -x
-            pwd
-#            docker-compose -f "docker-compose.yml" up -d --build
+            cd client/web
+            yarn install --silent
+            yarn global add react-scripts --silent
```

Comment:
You should use the one from node_module.
Currently you may use a version different from the one specified in the package.json...

Reply:
Ok, changed to use - image: circleci/node:13.12.0